src/XenCert/StorageHandler.py

```diff
@@ -1836,20 +1835,40 @@ def deleteVdiFromMetadata(self, vdi_uuid):
     
     def Probe_SR(self):
         return ''
-        
-class StorageHandlerISCSI(StorageHandler):
+
+
+class BlockStorageHandler(StorageHandler):
+
+    def __init__(self, storage_handler):
+        super(BlockStorageHandler, self).__init__(storage_handler)
+        self.gfs2_handler = StorageHandlerGFS2(self)
+
+    def ControlPathStressTests(self):
+        (retValControl, checkPointsControl, totalCheckPointsControl) = \
+            super(BlockStorageHandler, self).ControlPathStressTests()
+
+        if retValControl:
```

Comment:
We should probably put a version check in here. There doesn't appear to be any prior examples of this but I guess it can just check the local xapi.
There isn't any reason why the GFS2 test should be gated on the success of the LVM. I'll change this if to be the version check.